config/govuk_index/migrated_formats.yaml

```diff
@@ -64,7 +64,6 @@ migrated:
   - /world/afghanistan
   - /world/albania
   - /world/algeria
-  - /world/american-samoa
```

Comment:
How does this change result in the page being redirected?

Reply:
It doesn't. That's done in the admin interface of Content Tagger.
https://docs.publishing.service.gov.uk/manual/rename-a-country.html#2-update-worldwide-taxons
I probably should have written it in the description.

Reply:
Well, I'm looking at what the commit message says is being done: "Remove American Samoa to redirect it to USA"

If you're saying it doesn't do that, what does this change do?

Reply:
Let me know if it's clear now.

Reply:
Ok, yeah, that makes sense :+1: